issue 1:
 the python-magic installed, but unable to be called by the 